### PR TITLE
chore: ADD stylelint

### PR DESCRIPTION
**Pull request by sarovin**
Hi,
I want to add stylelint before starting with styled-system & add stylelint and eslint to husky hook.

### CODE REVIEW

**Comment by evertonfraga** [comment]
I just renamed the PR, so it adheres with Semantic Pull Request guidelines. 

Thank you for the contribution!

File: package.json [hunk 1]
```diff
@@ -89,8 +95,10 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,json,css,md}": [
-      "prettier --no-semi --single-quote --write",
+    "src/**/*.{js,jsx}": [
```
Comment (marcgarreau) [inline package.json]: I'd like to continue to apply prettier to multiple file types (markdown, etc.). `lint-staged` can accept multiple comma-separated tasks, so how about breaking it up? something like:
```
"lint-staged": {
    "*.{js,jsx,json,css,md}": "prettier --no-semi --single-quote --write",
    "src/**/*.{js,jsx}": [
        "yarn lint",
        "yarn lint:style"
    ],
    "*": "git add"
}
```

Comment (sarovin) [reply package.json]: Of course @marcgarreau, this is a good solution.
I have added a commit with this.

File: src/components/Widgets/Checkbox.jsx [hunk 2]
```diff
@@ -34,10 +34,10 @@ const StyledCheckbox = styled.input`
   position: relative;
   margin: 0;
   outline: none !important;
-  -webkit-appearance: none;
```
Comment (marcgarreau) [inline src/components/Widgets/Checkbox.jsx]: Heads up: we lost styling in Firefox here (I'm only checking Chrome and Firefox atm). Although not a valid regular css key, styled-components resolves this by accepting `appearance: none;` and adding the appropriate prefixes. I'll take care of this.

**Pull request merged**